README.md

```diff
@@ -39,79 +39,96 @@ const client = new ApolloClient({
 ```
 
 ## Options
-```js
-const defaultOptions = {
+```typescript
+export interface ApolloLinkSentryOptions {
+  /**
+   * Determines if the given operation should be handled or discarded.
+   *
+   * If undefined, all operations will be included.
+   */
+  shouldHandleOperation: undefined | ((operation: Operation) => boolean);
+
+  /**
+   * Set the Sentry transaction name to the GraphQL operation name.
+   *
+   * May be overwritten by other parts of your app.
+   */
+  setTransaction: true | false;
+
   /**
-   * Set the Sentry `transaction` to the `operationName` of the query / mutation. Note that this
-   * only works if the transaction is not overwritten later in your app.
+   * Narrow Sentry's fingerprint by appending the GraphQL operation name to the {{default}} key.
+   *
+   * Only the last executed operation will be added, not every operation that's been through the link.
+   * May be overwritten by other parts of your app.
    */
-  setTransaction: true,
+  setFingerprint: true | false;
 
   /**
-   * Narrow Sentry's fingerprint by appending the operation's name to Sentry's {{default}} key.
-   * It works in such a way that only the last operation is added, not every operation that's been
-   * through the link. Note that if you override this somewhere else in your app, it is possible
-   * that the value set by `apollo-link-sentry` is overwritten.
+   * Attach a breadcrumb for executed GraphQL operations.
+   *
+   * The following information will be included by default:
+   * {
+   *   type: 'http',
+   *   category: `graphql.${operationType}`,
+   *   message: operationName,
+   *   level: errors ? 'error' : 'info',
+   * }
    */
-  setFingerprint: true,
+  attachBreadcrumbs: AttachBreadcrumbsOptions | false;
```

Comment:
Perhaps this should also be singular?

Reply:
I guess an error contains multiple breadcrumbs, but whatever you want I'm also fine with.

Reply:
Yeah, i think this reads more natural. Thanks for the feedback.